arangod/GeneralServer/H2CommTask.cpp

```diff
@@ -659,7 +659,7 @@ void H2CommTask<T>::sendResponse(std::unique_ptr<GeneralResponse> res,
   // streams)
   unsigned retries = 512;
   try {
-    while (ADB_UNLIKELY(!_responses.push(tmp) && --retries > 0)) {
+    while (ADB_UNLIKELY_DEPRECATED(!_responses.push(tmp) && --retries > 0)) {
```

Comment:
Let's remove the deprecated macro. Not repeating this for the following occurrences.